src/translations/en.json

```diff
@@ -1004,8 +1004,8 @@
               "ping": "Reload ping binary sensor entities",
               "filesize": "Reload file size entities",
               "telegram": "Reload telegram notify services",
-              "smtp": "Reload smtp notify services",
-              "mqtt": "Reload manually configured mqtt entities",
+              "smtp": "Reload SMTP notify services",
```

Comment:
We should actually keep this one. My `titleCase()` method in the navigation commands PR will treat "smtp" as a normal word so it'd end up as "Reload Smtp Notify Services" if you don't change it here.

Reply:
IMHO since those are protocols/acronyms I think they should actually be all uppercase (see a few lines above "IP"). So perhaps your title method could check if a word is already all uppercase and then leave it that way?

Reply:
I think some signals got crossed :) You should definitely keep this change.

I was just saying that _if_ you removed this change, we'd end up with "Reload Smtp Notify Services" which is bad.

If you keep the change, well, `titleCase` is just upper-casing the first letter of each word, so we'd end up with "Reload SMTP" which is correct.